include/circt/Dialect/RTL/Structure.td

```diff
@@ -61,6 +58,57 @@ def RTLModuleOp : RTLOp<"module",
   let parser = [{ return ::parse$cppClass(parser, result); }];
 }
 
+def RTLExtModuleOp : RTLOp<"extmodule",
+      [IsolatedFromAbove, FunctionLike, Symbol]> {
+  let summary = "RTL extModule";
+  let description = [{
+    The "rtl.extmodule" operation represents an external reference to a Verilog
+    module, including a given name and a list of ports.
+  }];
+  let arguments = (ins OptionalAttr<StrAttr>:$defname,
```

Comment:
Does the RTL dialect need a `defname` thing here?  I think that is just a firrtl oddity.  If we need it, please add a summary about it to the description block.  Also, should it be optional?

Reply:
I don't what a `defname` is. Is is the name of the external module in case the MLIR symbol isn't sufficient?

Reply:
(Thanks to @jackkoenig for explaining this...) `defname` is a FIRRTL artifact that exists because FIRRTL doesn't represent parameterized modules. So, `defname` is a mechanism to map multiple external modules (which have concrete Verilog parameter arguments) back to the same Verilog module. 

I think we avoid this in RTL dialect if we have a mechanism to represent parameterized modules and passing parameters to instantiations of modules.

Reply:
Got it.

Regarding parameterized modules in RTL: see https://llvm.discourse.group/t/continuing-the-conversation-on-per-module-versus-per-instance/1807/6?u=jdd.